test/sharness/t0170-dht.sh

```diff
@@ -7,10 +7,11 @@ test_description="Test dht command"
 # start iptb + wait for peering
 NUM_NODES=5
 test_expect_success 'init iptb' '
-  iptb init -n $NUM_NODES --bootstrap=none --port=0 &&
-  startup_cluster $NUM_NODES
+  iptb init -n $NUM_NODES --bootstrap=none --port=0
 '
 
+startup_cluster $NUM_NODES
```

Comment:
Why remove this from the test? We're expecting success from this command, right?